stix_shifter/stix_translation/src/modules/carbonblack/carbonblack_data_mapping.py

```diff
@@ -0,0 +1,32 @@
+from os import path
+import json
+from ...exceptions import DataMappingException
+
+
+class CarbonBlackDataMapper:
+    def __init__(self, options):
+        self.map_data = self._fetch_mappings()
```

Comment:
I know in QRadar we first look for a mapping in `options['mapping']` in case the end-user want to override mappings on the data source. Do you think similar support is needed here? I realize by seeing this that I failed to add anything covering options in our documentation.

Reply:
I can add support for that back in. Due to the binary and process api having separate mappings it seemed weird to me a user overriding them.